spec/util/miq_object_storage_spec.rb

```diff
@@ -0,0 +1,121 @@
+require "fileutils"
+require "util/mount/miq_generic_mount_session"
+require "util/miq_object_storage"
+
+class MockLocalFileStorage < MiqObjectStorage
+  def initialize source_path = nil, byte_count = 2.megabytes
+    @byte_count   = byte_count
+    @source_input = File.open(source_path, "rb") if source_path
+    @root_dir     = Dir.tmpdir
+  end
+
+  def mkdir(dir)
+    FileUtils.mkdir_p(File.join(@root_dir, dir))
+  end
+end
+
+describe MiqObjectStorage do
+  describe "#write_single_split_file_for (private)" do
+    include_context "generated tmp files"
+
+    subject         { MockLocalFileStorage.new source_path }
+    let(:dest_path) { Dir::Tmpname.create("") { |name| name } }
+
+    it "copies file to splits" do
+      expected_splitfiles = (1..5).map do |suffix|
+        "#{dest_path}.0000#{suffix}"
+      end
+
+      expected_splitfiles.each do |file|
+        subject.send(:write_single_split_file_for, File.open(file, "wb"))
+      end
+
+      expected_splitfiles.each do |filename|
+        expect(File.exist?(filename)).to be true
+        expect(Pathname.new(filename).lstat.size).to eq(2.megabytes)
+      end
+    end
+
+    context "with slightly a slightly smaller input file than 10MB" do
+      let(:tmpfile_size) { 10.megabytes - 1.kilobyte }
+      subject            { MockLocalFileStorage.new source_path, 1.megabyte }
+
+      it "properly chunks the file" do
+        expected_splitfiles = (1..10).map do |suffix|
+          "#{dest_path}.%<suffix>05d" % {:suffix => suffix}
+        end
+
+        expected_splitfiles.each do |file|
+          subject.send(:write_single_split_file_for, File.open(file, "wb"))
+        end
+
+        expected_splitfiles[0, 9].each do |filename|
+          expect(File.exist?(filename)).to be true
+          expect(File.size(filename)).to eq(1.megabytes)
+        end
+
+        last_split = expected_splitfiles.last
+        expect(File.exist?(last_split)).to be true
+        expect(File.size(last_split)).to eq(1.megabyte - 1.kilobyte)
+      end
+    end
+
+    context "non-split files (byte_count == nil)" do
+      subject          { MockLocalFileStorage.new source_path, byte_count }
+
```

Comment:
I think if you're going to align the braces then there shouldn't be a new line here.

Reply:
Fixed.